packages/expo-sensors/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "expo-sensors",
-  "version": "11.4.0",
+  "version": "11.4.1",
```

Please don't bump version here, we'll do it during release
```suggestion
  "version": "11.4.0",
```